### DISCUSSION TITLE
Help on making aws-actions/configure-aws-credentials@v1 work with act?

### DISCUSSION THREAD

**Discussion by JeromeDuboisVizzia** [nektos/act#1346] [Q&A]
Dear community,

I am very fond of **act**, a smart tool to run locally GH Actions.
It really helped me debug Actions faster, so it is really a super cool tool.

However, I am using **aws-actions/configure-aws-credentials@v1** in my job and cannot make it work, so said job with this step fails.

Has anyone already made it work in act docker environment?
Any advice?

Thanks for your help!

Best,
Jérôme.

**Comment by ChristopherHX** [comment]
I have absolutly no idea how you are using that actions, but **act cannot use the feature `role-to-assume`**.

See here for the other two options, which might be compatible with act
https://github.com/aws-actions/configure-aws-credentials#assuming-a-role

The following modes can possible work in act, but I never did that myself
- aws-access-key-id
- web-identity-token-file

If you want to store those secrets in a `.secrets` file take attention that you have to escape newlines and use quotes if you have multiline secrets.
https://github.com/nektos/act/wiki/Beginner's-guide#envsecrets-files-structure
